### PR TITLE
Reference Cloudflare workers type by import

### PR DESCRIPTION
**Pull request by zhuhaow**
This avoids conflicts between "DOM" and @cloudflare/workers-types. See https://github.com/cloudflare/workers-types/issues/164

- [x] I have verified my changes didn't break the `example` project.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -8,8 +8,13 @@
...
-}
+  "include": [
+    "src"
+  ],
+  "exclude": [
+    "node_modules",
+    "dist",
+    "test"
+  ]
+}
```
Comment (aidenwallis) [inline tsconfig.json:20]: newline missing here

File: example/package.json [hunk 2]
```diff
@@ -17,4 +17,4 @@
     "kysely": "^0.23.3",
     "kysely-d1": "^0.2.0"
   }
-}
+}
```
Comment (aidenwallis) [inline example/package.json:20]: newline missing here

**Comment by aidenwallis** [comment]
This PR fails with the prettier check, can you run `npm run format` on your branch and push those changes? Thanks!

File: example/package-lock.json [hunk 3]
```diff
@@ -9,14 +9,28 @@
       "version": "0.0.0",
       "dependencies": {
         "kysely": "^0.23.3",
-        "kysely-d1": "^0.2.0"
+        "kysely-d1": "file:../"
```
Comment (aidenwallis) [inline example/package-lock.json:12]: How did you get this to happen? This shouldn't be the case.

Comment (zhuhaow) [reply example/package-lock.json]: Uhh, I tested the example but forgot to run "npm install" after change the "package.json" back.

Comment (aidenwallis) [reply example/package-lock.json]: Ah, got it, yeah, this is deliberate - I want the example project to be something people can just copy/paste the directory from and get started. 👍

**Comment by aidenwallis** [comment]
Given that this is a major bump with a sub-dependency, and cloudflare don't seem to be offering super stable types for D1 right now, I'm going to version this as a major release in case we nuke packages that depend on us. 👍

**Review by aidenwallis — APPROVED** [review]

**Comment by zhuhaow** [comment]
I think the lock file in the example is still a little problematic. Checking

**Comment by aidenwallis** [comment]
Worked for me, I can fix this in the release branch. 👍

**Comment by zhuhaow** [comment]
It should work since it's referencing the parent folder from the example. Just not sure why.

**Comment by aidenwallis** [comment]
It'll be sorted in the release branch. 👍

**Comment by zhuhaow** [comment]
Awesome, thanks.

**Pull request merged**